tests/testapps/CTCatalog/CTCatalog/Tests/CTCParagraphStyleTestViewController.mm

```diff
@@ -0,0 +1,507 @@
+PICKER.delegate = self;                                                                                               \
+PICKER.dataSource = self;                                                                                             \
+cell.accessoryView = PICKER;                                                                                          \
+[ARRAY addObject:cell];
+
+#define ADD_TEXT_FIELD(ARRAY, FIELD, TITLE, DEFAULT_VALUE)                                                            \
+cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:@"SHARED_PARAGRAPH_INPUT"]; \
+cell.textLabel.text = TITLE;                                                                                          \
+FIELD = [[UITextField alloc] initWithFrame:CGRectMake(0, 10, 200, 30)];                                               \
+FIELD.text = DEFAULT_VALUE;                                                                                           \
+FIELD.delegate = self;                                                                                                \
+cell.accessoryView = FIELD;                                                                                           \
+[ARRAY addObject:cell];
+// clang-format on
+
+@interface CTParagraphStyleTestView : UIView {
+}
+
+@property (nonatomic) CTParagraphStyleRef paragraphStyle;
+
+@end
+
+@implementation CTParagraphStyleTestView {
+}
+
+- (void)drawRect:(CGRect)rect {
+    UIColor* color = [UIColor blueColor];
+
+    CGContextRef context = UIGraphicsGetCurrentContext();
+
+    // Aligns origin for our frame
+    CGContextTranslateCTM(context, 0.0f, self.bounds.size.height);
+
+    // Flips y-axis for our frame
+    CGContextScaleCTM(context, 1.0f, -1.0f);
+
+    // Creates path with current rectangle
+    CGMutablePathRef path = CGPathCreateMutable();
+    CGPathAddRect(path, NULL, self.bounds);
+
+    UIFont* font = [UIFont systemFontOfSize:20];
+    CTFontRef myCFFont = CTFontCreateWithName((__bridge CFStringRef)[font fontName], [font pointSize], NULL);
+    // Make dictionary for attributed string with font, color, and alignment
+    NSDictionary* attributesDict = [NSDictionary dictionaryWithObjectsAndKeys:(__bridge id)myCFFont,
+                                                                              (id)kCTFontAttributeName,
+                                                                              color.CGColor,
+                                                                              (id)kCTForegroundColorAttributeName,
+                                                                              (__bridge id)_paragraphStyle,
+                                                                              (id)kCTParagraphStyleAttributeName,
+                                                                              nil];
+
+    NSString* text = @"The quick brown fox jumps over the lazy dog.";
```

Comment:
It looks like they're both in the same family Canidae but different species.

---
In reply to: [76319688](https://github.com/Microsoft/WinObjC/pull/909#discussion_r76319688) [](ancestors = 76319688)